trade-finance-manager-ui/server/controllers/case/amendments/underwriting/amendment-underwriter-managers-decision/index.js

```diff
@@ -0,0 +1,120 @@
+const api = require('../../../../../api');
+
+const { canUserEditManagersDecision } = require('../helpers');
+// const { validateSubmittedValues } = require('./validateSubmittedValues');
+// const { mapDecisionObject } = require('./mapDecisionObject');
+
+/**
+ * @param {Object} deal
+ * @param {Object} amendment
+ * @param {Object} user
+ * @returns {Object}
+ * checks if user can edit managers decision and returns object
+ */
+const getAmendmentUnderwriterManagersDecision = async (amendment, user) => {
+  const isEditable = canUserEditManagersDecision(
+    user,
+    amendment,
+  );
+
+  return {
+    isEditable,
+    dealId: amendment.dealId,
+    facilityId: amendment.facilityId,
+    amendmentId: amendment.amendmentId,
+    underwriterManagersDecision: amendment.underwriterManagersDecision,
+    amendment,
+  };
+};
+
+/**
+ * @param {*} req
+ * @param {*} res
+ * renders first page of amendment managers decision if can be editted by user
+ */
+const getAmendmentUnderwriterManagersDecisionEdit = async (req, res) => {
+  const { _id: dealId, amendmentId, facilityId } = req.params;
+  const deal = await api.getDeal(dealId);
+
+  const { data: amendment } = await api.getAmendmentById(facilityId, amendmentId);
+
+  if ((!amendment || !amendment.amendmentId) || !deal) {
```

Comment:
not a show stopper for this pr, but try to keep this consistent with the other files. we are using a combination of !amendment, !amendment.amendmentId or !deal